projects/plugins/jetpack/_inc/client/state/at-a-glance/actions.js

```diff
@@ -167,3 +170,25 @@ export const fetchPluginUpdates = () => {
 			} );
 	};
 };
+
+export const fetchBackupUndoEvent = () => {
```

Comment:
As a follow-up task, we could consider moving these actions/reducers to the `js-packages` package.